### PR TITLE
Added Docker 1.13.x to the list of supported versions, with a caveat.

### PR DESCRIPTION
**Pull request by greggomann**
## Description
https://jira.mesosphere.com/browse/COPS-2166

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by greggomann** [comment]
@pavisandhu would you have a cycle to help review this?

File: pages/1.10/installing/ent/custom/system-requirements/index.md [hunk 1]
```diff
@@ -99,12 +99,13 @@ High speed internet access is recommended for DC/OS installation. A minimum 10 M
...
 Docker must be installed on all bootstrap and cluster nodes. The supported versions of Docker are:
 
+- 1.13.x
 - 1.12.x
 - 1.11.x
 
 **Recommendations**
 
-- Docker 1.11.x - 1.12.x is recommended <a href="https://github.com/docker/docker/issues/9718" target="_blank">for stability reasons</a>.
+- Docker 1.11.x - 1.13.x is recommended <a href="https://github.com/docker/docker/issues/9718" target="_blank">for stability reasons</a>. Note that due to an <a href="https://github.com/moby/moby/issues/33820" target="_blank">open issue in the moby project</a>, some users may experience Docker containers stuck in an unresponsive state when using version 1.13.x.
```
Comment (vinodkone) [inline pages/1.10/installing/ent/custom/system-requirements/index.md]: I would rephrase this as follows:

-- Docker 1.11x - 1.12.x is recommended <> for stability reasons.
-- Docker 1.13.x has a known issue <> which may result in some users experiencing Docker containers getting stuck in unresponsive state.
